src/utils/download.ts

```diff
@@ -30,6 +31,7 @@ export async function downloadProject(project: Project, uri?: string) {
   }
   const queue: Array<{filePrefix: string; file: File}> = [];
   project.mapEachFile((f: File) => queue.push({filePrefix: "", file: f}));
+  await addDevKey(queue, uri);
```

Comment:
I think it's better to do this as:
```
queue.push(await getDevKey(uri));
```

Makes for easier to test and more flexible method

Reply:
getDevKey may return null though